### PR TITLE
Change Logo

### PR DESCRIPTION
**Pull request by GeetikaBatra**
Signed off by: Geetika Batra <gbatra@redhat.com>

### CODE REVIEW

**Comment by GeetikaBatra** [comment]
@invincibleJai. PTAL

**Comment by GeetikaBatra** [comment]
[test]

File: README.adoc [hunk 1]
```diff
@@ -1,6 +1,15 @@
...
+Red Hat OpenShift.io services extension allows developers using more traditional development tools to benefit from the security, license analysis and AI based suggested dependencies that [OpenShift.io](https://openshift.io/) provides:
+- Common Vulnerabilities and Exposures (CVE) analysis highlights code dependencies that have open CVEs against them.
+- License analysis determines the license used by the project and its dependencies. If the project already has a license the analysis will show any conflicts between that license and dependency licenses that may be more restrictive. If the project does not have a license, OpenShift.io will suggest a license for the project that is sufficiently permissive to match the license restrictions of the dependencies.
+- AI based suggested dependencies that can be added to your application stack, alternative dependencies to currently used dependencies if any of the currently used dependencies are not typically used together in an application stack
 
+Today OpenShift.io services extension supports projects using Maven and projects build on npm (Node ecosystem). Extending support for Python and Go languages is currently under progress.
```
Comment (invincibleJai) [inline README.adoc]: is NPM/node supported? and are we going to extend it to python and go

File: README.adoc [hunk 2]
```diff
@@ -1,6 +1,15 @@
...
+
+= Fabric8Analytics Devstudio Plugin
+
+This plugin adds support of Fabric8Analytics in Devstudio.
 TODO: write introduction to this project & its purpose
```
Comment (invincibleJai) [inline README.adoc]: can we have what is needed for TODO

File: plugins/com.redhat.fabric8analytics.lsp.eclipse.ui/plugin.xml [hunk 3]
```diff
@@ -83,7 +83,7 @@
                id="com.redhat.fabric8analytics.lsp.eclipse.ui.Authorize">
             <command
                   commandId="com.redhat.fabric8analytics.lsp.eclipse.ui.authorizetoken"
-                  icon="icons/osio-16x16.png"
+                  icon="icons/openshift_logo.png"
```
Comment (invincibleJai) [inline plugins/com.redhat.fabric8analytics.lsp.eclipse.ui/plugin.xml]: check how openshift_logo is showing up as you need 16*16 looks like

File: README.adoc (hunk 1 above)
Comment (GeetikaBatra) [reply README.adoc]: It doesn't, but this statement is regarding Openshift.io service and not directed to Fabric8-analytics-devstudio plugin. So kept it as it is.

File: plugins/com.redhat.fabric8analytics.lsp.eclipse.ui/plugin.xml (hunk 3 above)
Comment (GeetikaBatra) [reply plugins/com.redhat.fabric8analytics.lsp.eclipse.ui/plugin.xml]: can't check on my system, right now. It crashed it. Maybe once I get the new system. :)

File: README.adoc (hunk 2 above)
Comment (GeetikaBatra) [reply README.adoc]: removing this, as intro is covered above.

File: README.adoc (hunk 1 above)
Comment (invincibleJai) [reply README.adoc]: i think otherwise @sivaavkd can update

**Comment by GeetikaBatra** [comment]
Merging this one, please create an issue if you still feel like the ReadMe needs a change.

**Pull request merged**